Neos.Neos/Classes/Service/EditorContentStreamZookeeper.php

```diff
@@ -102,48 +102,39 @@ public function relayEditorAuthentication(Authentication\TokenInterface $token):
-            if ($user instanceof User) {
-                $workspaceName = AdjustmentsWorkspaceName::fromAccountIdentifier(
-                    $token->getAccount()->getAccountIdentifier()
-                );
-                $workspace = $contentRepository->getWorkspaceFinder()->findOneByName(
-                    $workspaceName->toContentRepositoryWorkspaceName()
-                );
-
-                if (!$workspace) {
-                    // @todo: find base workspace for user
-                    /** @var Workspace $baseWorkspace */
-                    $baseWorkspace = $contentRepository->getWorkspaceFinder()->findOneByName(WorkspaceName::forLive());
-                    $editorsNewContentStreamId = ContentStreamId::create();
-                    $similarlyNamedWorkspaces = $contentRepository->getWorkspaceFinder()->findByPrefix(
-                        $workspaceName->toContentRepositoryWorkspaceName()
-                    );
-                    if (!empty($similarlyNamedWorkspaces)) {
-                        $workspaceName = $workspaceName->increment($similarlyNamedWorkspaces);
-                    }
-
-                    $contentRepository->handle(
-                        CreateWorkspace::create(
-                            $workspaceName->toContentRepositoryWorkspaceName(),
-                            $baseWorkspace->workspaceName,
-                            new WorkspaceTitle((string) $user->getName()),
-                            new WorkspaceDescription(''),
-                            $editorsNewContentStreamId,
-                            UserId::fromString($this->persistenceManager->getIdentifierByObject($user))
-                        )
-                    )->block();
-                } else {
-                    CatchUpTriggerWithSynchronousOption::synchronously(fn() =>
-                        $contentRepository->handle(
-                            RebaseWorkspace::create(
-                                $workspace->workspaceName,
-                            )
-                        )->block());
-                }
-            }
+        if (!$isEditor) {
+            return;
+        }
+        $user = $this->partyService->getAssignedPartyOfAccount($token->getAccount());
+        if (!$user instanceof User) {
+            return;
         }
+        $workspaceName = WorkspaceNameBuilder::fromAccountIdentifier(
+            $token->getAccount()->getAccountIdentifier()
+        );
+        $workspace = $contentRepository->getWorkspaceFinder()->findOneByName($workspaceName);
+        if ($workspace !== null) {
+            CatchUpTriggerWithSynchronousOption::synchronously(fn() => $contentRepository->handle(
+                RebaseWorkspace::create(
+                    $workspace->workspaceName,
+                )
+            )->block());
+            return;
+        }
+
+        // @todo: find base workspace for user
```

Comment:
Tiny nitpick: Is this really still a todo? We are creating a new user workspace if we couldn't find one. And this would always be based on the Live workspace before. Wouldn't it?

Reply:
I don’t know if that behaviour was customizeable in 8.x

Reply:
basti [wrote](https://github.com/neos/neos-development-collection/pull/4534#discussion_r1339644745)

> Let's remove it. If it is needed, we'll find out and can add it :)